seatunnel-e2e/seatunnel-e2e-common/src/test/java/org/apache/seatunnel/e2e/common/container/flink/Flink18Container.java

```diff
@@ -0,0 +1,71 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.seatunnel.e2e.common.container.flink;
+
+import org.apache.seatunnel.e2e.common.container.TestContainer;
+import org.apache.seatunnel.e2e.common.container.TestContainerId;
+
+import com.google.auto.service.AutoService;
+import lombok.NoArgsConstructor;
+
+import java.io.File;
+
+/**
+ * This class is the base class of FlinkEnvironment test for new seatunnel connector API. The before
+ * method will create a Flink cluster, and after method will close the Flink cluster. You can use
+ * {@link Flink18Container#executeJob} to submit a seatunnel config and run a seatunnel job.
+ */
+@NoArgsConstructor
+@AutoService(TestContainer.class)
+public class Flink18Container extends AbstractTestFlinkContainer {
```

Comment:
Please update doc for flink 1.17 and 1.18
https://github.com/apache/seatunnel/blob/7b7f06437a62c1683f002db78458ec7dd90de108/docs/en/start-v2/locally/quick-start-flink.md#step-4-run-seatunnel-application

cc @TyrantLucifer any other doc/shell we should update?

Reply:
thank you for suggestion, I've modified it, also https://github.com/apache/seatunnel/blob/7b7f06437a62c1683f002db78458ec7dd90de108/docs/zh/start-v2/locally/quick-start-flink.md#step-4-run-seatunnel-application.